pipelines/rj_iplanrio__nf_agent/flow.py

```
@@ -0,0 +1,39 @@
+"""
+
+from prefect import flow
+
+
+@flow(log_prints=True)
+def nf_processing_flow(
+    bq_input_table: str | None = None,
+    bq_status_table: str | None = None,
+    batch_size: int = 1000,
+    gcs_output_base_path: str = "staging/brutos_poc_osinfo_ia/resultado_extracao_modelo",
+    db_path: str = "/tmp/nf_pipeline_cache.db",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/nf_pipeline_cache.db"

(S108)